docs/en/ingest-management/migrate-beats-to-agent.asciidoc

```diff
@@ -0,0 +1,513 @@
+data to {es}, add another integration. For guidance on which integrations you
+need, look at the list you created earlier when you
+<<prepare-for-migration,prepared for the migration>>.
+
+For example, if the agent policy you created earlier includes the System
+integration, but you also want to monitor Nginx:
+
+. From the main menu in {kib}, click *Add integrations* and add the Nginx
+integration.
++
+[role="screenshot"]
+image::images/migration-add-nginx-integration.png[Screen showing the Nginx integration]
+
+. Configure the integration, then apply it to the agent policy you used earlier.
+Make sure you expand collapsed sections to see all the settings like log paths.
++
+[role="screenshot"]
+image::images/migration-add-integration-policy.png[Screen showing Nginx configuration]
++
+When you deploy your changes, the agent policy is updated to include a new
+integration policy for Nginx. All {agent}s enrolled in the agent policy get the
+updated policy, and the {agent} running on your host will begin collecting Nginx
+data.
++
+NOTE: Integration policy names must be globally unique across all agent
+policies.
+
+. Go back to *{fleet} > Agents* and verify that the agent status is still
+healthy. Click the host name to drill down into the agent details. From there,
+you can see the agent policy and integration policies that are applied. 
++
+If the agent status is not Healthy, click *Logs* to view the agent log and
+troubleshoot the problem.
+
+. Go back to the main *{fleet}* page, and click *Data streams* to inspect the
+data streams and navigate to the pre-built dashboards installed with the
+integration.
+
+Notice again that the data is duplicated because you still have {beats}
+running and sending data.
+
+[discrete]
+== Migrate processor configurations
+
+Processors enable you to filter and enhance the data before it’s sent to the
+output. Each processor receives an event, applies a defined action to the event,
+and returns the event. If you define a list of processors, they are executed in
+the order they are defined. Elastic provides a
+{filebeat-ref}/defining-processors.html[rich set of processors] that are
+supported by all {beats} and by {agent}.
+
+Prior to migrating from {beats}, you defined processors in the configuration
+file for each Beat. After migrating to {agent}, however, the {beats}
+configuration files are redundant. All configuration is policy-driven from
+{fleet} (or for advanced use cases, specified in a standalone agent policy). Any
+processors you defined previously in the {beats} configuration need to be added
+to an integration policy; they cannot be defined in the {beats} configuration.
+
+IMPORTANT: Globally-defined processors are not currently supported by {agent}.
+You must define processors in each integration policy where they are required.
+
+To add processors to an integration policy:
+
+//REVIEWERS: I noticed that some integrations (like the System integration) do
```

Comment:
in some instances they are not applied t the whole policy rather being applied to a specific log. For example in the system integration each individual Windows event log has a section for processors.

Reply:
Yah I noticed that when I did my second round of testing.